cmd/root.go

```diff
@@ -68,7 +68,9 @@ func AddCommands(cmd *cobra.Command) {
 	/** ======================== P2P Flags ========================== **/
 
 	cmd.PersistentFlags().IntVar(&config.P2P.TCPPort, "tcp-port",
-		config.P2P.TCPPort, "TCP Port to listen on")
+		config.P2P.TCPPort, "inet port for P2P listener")
+	cmd.PersistentFlags().StringVar(&config.P2P.TCPInterface, "tcp-interface",
+		config.P2P.TCPInterface, "inet interface for P2P listener")
```

Comment:
this is the iface inet address right? I mean the iface name is probably "lo".

Reply:
It expects an IP address here, not an interface name, I think. Will double check this and make it clearer in the description if it must be an IP.